src/function_run_result.rs

```diff
@@ -57,7 +58,7 @@ impl fmt::Display for FunctionRunResult {
             formatter,
             "{}\n\n{}",
             "            Input            ".black().on_bright_yellow(),
-            self.input
+            serde_json::to_string_pretty(&self.input).unwrap_or_else(|error| error.to_string())
```

Comment:
```suggestion
            serde_json::to_string_pretty(&self.input).expect("Input should be serializable to a string")
```

This seems like a program failure if this is the case rather than an expected error condition.